### PR TITLE
Fixes #14052 - precompile assets

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by mmoll** [comment]
we need a new packaged foreman-tasks to fix the build error...

**Comment by ares** [comment]
ok, I will send it in separate PR

**Comment by ares** [comment]
depends on https://github.com/theforeman/foreman-packaging/pull/1052

**Comment by mmoll** [comment]
[test] again

File: plugins/ruby-foreman-chef/debian/rules [hunk 1]
```diff
@@ -9,5 +9,17 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
+PLUGIN = foreman_setup
```
Comment (mmoll) [inline plugins/ruby-foreman-chef/debian/rules]: foreman_chef :poultry_leg:

Comment (ares) [inline plugins/ruby-foreman-chef/debian/rules]: :scream_cat::

**Comment by mmoll** [comment]
cherry-picked as 6d63cf1 & 112f3d8 and as 0326ef9 & eb4dff5 for 1.11, díky @ares!

**Pull request closed without merging**